ui/imports/utils/Utils.qml

```diff
@@ -435,7 +435,9 @@ QtObject {
 
     function isEnsVerified(publicKey) {
         if (publicKey === "" || !isChatKey(publicKey) )
-            return
+            return false
+        if (!mainModuleInst)
```

Comment:
This is the fix of identicon ring

Reply:
this may be done in one line `return !mainModuleInst && mainModuleInst.isEnsVerified(publicKey)` :)